kalite/distributed/static/js/distributed/video/hbtemplates/video-player.handlebars

```diff
@@ -1,9 +1,13 @@
 <div id="video-player"{{#unless content_urls }} class="client-online-only"{{/unless}}>
     <video class="video-player video-js vjs-default-skin vjs-fullscreen" id="{{ random_id }}">
         <source src="{{ content_urls.stream }}" type="{{ content_urls.stream_type }}" />
-        {{#each subtitle_urls }}
-        <track kind="captions" src="{{ this.url }}" srclang="{{ code }}" label="{{ name }}" {{#ifcond code "==" ../selected_language }}{{#ifcond ../code "!=" "en" }}default="True"{{/ifcond}}{{/ifcond}}/>
-        {{/each}}
+        {{#current_language_is "en"}}
+            {{#ifcond translated_youtube_lang "==" "en"}}
```

Comment:
Hi @benjaoming and @mrpau-eugene build 0.17.x language packs are included subtitles.
If we check on the [Makefile](https://github.com/learningequality/ka-lite-content-packs/blob/master/Makefile) in the `ka-lite-content-packs`. We didn't specify `--no-subtitles ` args during building the language packs.
So by default it is downloading the subtitles.